### DISCUSSION TITLE
How do I build c++.

### DISCUSSION THREAD

**Discussion by 1Codealot** [raysan5/raylib#3147] [Q&A]
I am using the g++ compiler with sublime text but the raylib.sublime-build (which has 
```{
	"variants": [
		{ 
			"name": "desktop",
			"shell_cmd": "cd ${folder}\\src && mingw32-make PLATFORM=PLATFORM_DESKTOP"
		},

		{ 
			"name": "examples",
			"shell_cmd": "cd ${folder}\\examples && mingw32-make PLATFORM=PLATFORM_DESKTOP"
		},
	]
})
```
 
doesn't work. I get `The system canot find the path specified.`

Nvm. I used `g++ core_basic_window.cpp -lraylib -lopengl32 -lgdi32 -lwinmm` now i get random stuff in console that I don't want.

**Comment by orcmid** [comment]
Where are you finding `core_basic_window.cpp`?  Are you adapting `core_basic_window.c` somehow?

Are you talking about some sort of log output to the console window?

**Reply by 1Codealot** [reply]
Yes im talking about the logs. Is there a way to get rid of it? Also the only change I made was renaming `core_basic_window.c` to `core_basic_window.cpp`

**Reply by orcmid** [reply]
See what happens if you rename it back to `core_basic_window.c`.  GCC won't mind.

This may have something to do with how `lraylib` was built.

You can also redirect the console output to NULL on the command to run your executable, assuming you are getting the basic window to open and show you its message.

**Comment by Peter0x44** [comment]
> now i get random stuff in console that I don't want.


It's not random, assuming you're referring to the tracelog there.
For some situations, it is very useful output. I would keep it on during development, always.
Anyway, it's documented how to disable the console on the raylib wiki, right here:
https://github.com/raysan5/raylib/wiki/Frequently-Asked-Questions#how-do-i-remove-the-console-window
